parm/config/gfs/config.ufs

```diff
@@ -289,22 +285,25 @@ case "${fv3_res}" in
           export tau=3.0
           export rf_cutoff=100.0
           export fv_sg_adj=450
-          export WRITE_GROUP_GDAS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=15
-          export WRITE_GROUP_GFS=4
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=20 #Note this should be 10 for WCOSS2
         fi
         ;;
     "C1152")
         export DELTIM=150
-        export layout_x_gdas=8
-        export layout_y_gdas=16
-        export layout_x_gfs=8
-        export layout_y_gfs=16
-        export nthreads_fv3_gdas=4
-        export nthreads_fv3_gfs=4
-        export nthreads_ufs_gdas=4
-        export nthreads_ufs_gfs=4
+        if [[ "${RUN}" = *gdas ]]; then
+          export layout_x=8
+          export layout_y=16
+          export nthreads_fv3=4
+          export nthreads_ufs=4
+          export WRITE_GROUP=4
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=20 # TODO: refine these numbers when a case is available
+        elif [[ "${RUN}" = *gfs ]]; then
+          export layout_x=8
+          export layout_y=16
+          export nthreads_fv3=4
+          export nthreads_ufs=4
+          export WRITE_GROUP=4
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=20 # TODO: refine these numbers when a case is available
```

Comment:
I think these options are likely to change for `gdas` and `gfs`, so I'm OK with this if-elif block, even though the variables are the same.